src/main/java/com/amazon/reviews/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.amazon.reviews.service.impl;
+
+import com.amazon.reviews.exception.FileProcessingException;
+import com.amazon.reviews.service.FileReaderService;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FileReaderServiceImpl implements FileReaderService {
+
+    @Override
+    public List<String> read(String path) {
+        try (Stream<String> stream = Files.lines(Paths.get(path))) {
```

Comment:
I think we should better move this out from try-with-resources
and just use try-catch)